CONTRIBUTING.md

```diff
@@ -0,0 +1,230 @@
+If this is your first time editing,
+you will see with a page asking you to fork the repository before being able to edit.
+Click through the link to do this,
+and GitHub will create a copy of the entire repository linked to your own account.
+Feel free to edit any page in this repository,
+as your changes won't reflection the main repository or the Read the Docs site until you make a pull request.
+
+#### Editing the page
+
+The documentation pages are written in two different formats,
+reStructuredText (RST) and Markdown (MD).
+You can see which one of these the file you are editing is by checking its file extension,
+above the content window.
+You can find advice on formatting text [here for `.rst`]
+and [here for `.md`].
+For tasks such as correcting some information or fixing a typo,
+you don't need to worry about the formatting.
+
+Once you've made changes,
+to save them you will need to "commit" them.
+Click on the green "commit changes" button,
+and choose to either commit to your main branch or make a new branch,
+if you wish to make more complicated changes.
+When you're happy with all your changes
+and wish to update the main repository for the Read the Docs site,
+you can make a pull request (PR).
+
+  [here for `.rst`]: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
+  [here for `.md`]: https://www.markdownguide.org/basic-syntax/
+
+### Making a pull request
+
+To Make a pull request,
+first click on the "Pull requests" tab at the top of GitHub.
+From here, click the green "New pull request" button,
+which should take you to a page comparing the main repository to your fork.
+You should see any commits you have made listed here.
+Clicking "Create pull request" will give you the opportunity to give your edits a title
+and a brief description,
+before you submit them for review.
+
+At this point,
+a maintainer of the repository will be able to review your changes
+to ensure they're sensible and don't break anything.
+If all is well, they will approve the changes and merge them into the main repo.
+After the reviewer has merged the pull request,
+Read the Docs will recompile the page and publish your changes.
+
+## Making a contribution
+
+### Structure
+
+This documentation follows the [Diátaxis] documentation framework.
+We recommend reading the [Diátaxis] documentation.
+
+What this means for the EPICS documentation,
+is that documentation falls into 4 categories:
+
+-   Tutorials: for users to learn new concepts
+-   How-to guides: for users to achieve specific, predefined goals
+-   References: for users to consult,
+    when looking for information
+-   Explanations: for users to clarify their understanding of a concept
+
+TODO: explain where users should contribute, when we figure out if we merge every documentation in the same place.
+
+  [Diátaxis]: https://diataxis.fr/
+
+### GitHub requirements
+
+To propose your changes,
+you will need a GitHub account,
+and a fork under your name.
```

Comment:
Pedantic, but it might be wise to say something like "fork into your namespace" or "fork as a personal project" or something like that, as it is not your _name_ that you are forking it under.